apps/web/components/booking/BookingListItem.tsx

```diff
@@ -93,6 +94,24 @@ function BookingListItem(booking: BookingItemProps) {
     mutation.mutate(body);
   };
 
+  const getSeatReferenceUid = () => {
+    const attendee = booking.attendees.find((attendee) => {
+      if (attendee.email === user?.email) return attendee;
+    });
+
+    console.log("🚀 ~ file: BookingListItem.tsx:99 ~ getSeatReferenceUid ~ attendee", attendee);
+
+    if (attendee) {
+      const attendeeSeatReference = booking.seatsReferences.find(
+        (attendeeSeatReference) => attendeeSeatReference.attendeeId === attendee.id
+      );
+
+      if (attendeeSeatReference) return attendeeSeatReference.referenceUId;
+    }
+
+    return "";
+  };
+
```

Comment:
We will pass the `seatReferenceUId` to identify which attendee we should remove from the booking